lib/Conch/Controller/DeviceReport.pm

```diff
@@ -228,7 +231,7 @@ sub _record_device_report {
 			my %inactive_serials = map { $_->serial_number => 1 } @device_disks;
 
 			foreach my $disk ( keys %{ $dr->{disks} } ) {
-				$log->debug("Device $device_id: Recording disk: $disk");
+				$log->debug("Device $device->id: Recording disk: $disk");
```

Comment:
This will not render correctly. use `$device_id`.

Reply:
Addressed